.chloggen/1421.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: breaking
+component: database
+note: Add new `db.response.status_code` attribute, deprecate `db.cosmos.status_code`.
```

Comment:
Are we able to express this in schema migrations or not?

Reply:
unfortunately no, it went from int to string